docs/LPC_Additional_LPC_Cycles.md

```diff
@@ -0,0 +1,248 @@
+## Why do we need to support additional LPC protocol cycles in "LPC Host"
+
+In the implementation of the LPC (LPC Host and LPC Peripheral) protocol we wrote
+earlier, only two types of LPC cycles were supported:
++ I / O cycles (Read and Write)
++ TPM cycles
+
+And we really do not need to support more types of LPC cycles in the designed 
```

Comment:
Actually, we need only TPM cycles, not I/O. But those only differ in `START` field.